integration/js/VideoTestSimulcast.js

```diff
@@ -0,0 +1,42 @@
+const {OpenAppStep, JoinMeetingStep, AuthenticateUserStep, ClickVideoButton, WaitForRemoteVideoCheckToComplete, WaitForRemoteParticipantsToTurnVideoOff, WaitForRemoteParticipantsToTurnVideoOn, WaitForRemoteParticipantsToJoinMeeting, WaitForMeetingToBeCreated} = require('./steps');
+const {UserJoinedMeetingCheck, LocalVideoCheck, RemoteVideoCheck, UserAuthenticationCheck, RosterCheck} = require('./checks');
+const {AppPage} = require('./pages/AppPage');
+const {TestUtils} = require('./node_modules/kite-common');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const uuidv4 = require('uuid/v4');
+
+class VideoTestSimulcast extends SdkBaseTest {
+  constructor(name, kiteConfig) {
+    super(name, kiteConfig, "Video");
+  }
+
+  async runIntegrationTest() {
```

Comment:
Is this test basically the same as VideoTest.js? Could we instead create a new flag in the config file (similar to Selenium sessions in payload) and pass in a parameter to the VideoTest.js to decide whether to enable Simulcast or not?